crates/linear_algebra/src/orientation.rs

```diff
@@ -0,0 +1,74 @@
+use nalgebra::{RealField, SimdRealField};
+
+use crate::{Framed, Rotation2, Vector2, Vector3};
+
+pub type Orientation2<Frame, T = f32> = Framed<Frame, nalgebra::UnitComplex<T>>;
+pub type Orientation3<Frame, T = f32> = Framed<Frame, nalgebra::UnitQuaternion<T>>;
+
+impl<Frame, T> Orientation2<Frame, T>
+where
+    T: SimdRealField + Copy,
+    T::Element: SimdRealField,
+{
+    pub fn new(angle: T) -> Self {
+        Self::wrap(nalgebra::UnitComplex::new(angle))
+    }
+
+    pub fn identity() -> Self {
+        Self::wrap(nalgebra::UnitComplex::identity())
+    }
+
+    pub fn inverse(&self) -> Self {
```

Comment:
I don't think it makes sense to "invert" an orientation.

```suggestion
    pub fn mirror(&self) -> Self {
```

Reply:
I agree. Done.